pom.xml

```diff
@@ -99,10 +99,5 @@
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>credentials</artifactId>
     </dependency>
-    <dependency>
-      <groupId>org.mockito</groupId>
-      <artifactId>mockito-core</artifactId>
-      <scope>test</scope>
-    </dependency>
```

Comment:
Just for the record, this seems like a cleanup not related to JUnit 4 -> 5 migration.